thirdeye/pom.xml

```diff
@@ -22,17 +22,18 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.pinot.thirdeye</groupId>
   <artifactId>thirdeye</artifactId>
-  <version>${revision}${sha1}</version>
+  <version>0.1.0-SNAPSHOT</version>
   <packaging>pom</packaging>
   <name>thirdeye</name>
 
   <modules>
     <module>thirdeye-frontend</module>
+    <module>thirdeye-spi</module>
     <module>thirdeye-pinot</module>
   </modules>
 
   <properties>
-    <revision>1.0.0</revision>
+    <revision>0.1.0</revision>
```

Comment:
Why lower the version? Shouldn't this be `2.0.0`?

Reply:
I'm restoring the version changes. I think this will be driven more from a marketing standpoint. Calling it 2.0 simply raises expectations when we are not providing any additional functionality.